Sources/PeripheryKit/CodeRemoval/PublicAccessibilitySyntaxRemover.swift

```diff
@@ -1,13 +1,14 @@
 import Foundation
+import SourceGraph
 import SwiftParser
 import SwiftSyntax
 import SystemPackage
 
 final class PublicAccessibilitySyntaxRemover: SyntaxRewriter, SyntaxRemover {
-    private let resultLocation: SourceLocation
+    private let resultLocation: Location
```

Comment:
Please revert renaiming `SourceLocation` to `Location`.

Reply:
There is a problem with SourceLocation. SourceLocation was a structure in PeripheryKit, because of that Swift chose this realisation by default, but now by default it chose SwiftSyntax realisation. If we want to save SourceLocation name we need to write SwiftSyntax.SourceLocation,  SourceGraph.SourceLocation. What do you think is better to change the name or to write target name?